graph/color.go

```diff
@@ -51,7 +51,8 @@ func hsla(h, s, l, a float64) (r, g, b, ra float64) {
 	return
 }
 
-// sat8 converts 0..1 float to 0..255 uint8
+// sat8 converts 0..1 float to 0..255 uint8.
+// sat8 is short for saturate 8, referring to 8 byte saturation arithmetic
```

Comment:
"Saturation arithmetic" doesn't really tell much more than "sat". Maybe something like:

```
// sat8 converts 0..1 float to 0..255 uint8 with saturation.
//
//     sat8(x) = 0   if x < 0
//     sat8(x) = 255 if x > 1
```

Reply:
A reference to saturation arithmetic could help see that it's not something you came up with, but a broader approach. This also allows the documentation be shorter.

I like the `math` package like docs you added.